app/src/main/java/com/serwylo/lexica/game/Game.java

```diff
@@ -293,13 +314,37 @@ public void addWord(String word) {
 	}
 
 	public int getWordScore(String word) {
-		return WORD_POINTS[word.length()];
+		if (SCORE_WORDS.equals(scoreType)) {
+			return WORD_POINTS[word.length()];
+		} else {
+			word = word.toUpperCase();
+			int score = 0;
+			for (int i = 0; i < word.length(); i++) {
+				score += LETTER_POINTS[word.charAt(i) - 'A'];
+				if (word.charAt(i) == 'Q') i++;
```

Comment:
Not a huge fan of single line if statements, as they tend to be a source of bugs. Do you mind please replacing with:

```
if (word.charAt(i) == 'Q') {
    i++;
}
```